sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/filter/UserPrincipalManager.java

```diff
@@ -198,7 +197,7 @@ public void verify(JWTClaimsSet claimsSet, SecurityContext ctx) throws BadJWTExc
                 if (!isAadIssuer(issuer)) {
                     throw new BadJWTException("Invalid token issuer");
                 }
-                if (explicitAudienceCheck) {
```

Comment:
refer: https://sonarcloud.io/project/issues?directories=sdk%2Fspring%2Fspring-cloud-azure-autoconfigure%2Fsrc%2Fmain%2Fjava%2Fcom%2Fazure%2Fspring%2Fcloud%2Fautoconfigure%2Faad%2Ffilter&resolved=false&types=CODE_SMELL&id=stliu_azure-sdk-for-java&open=AXtAG3Fk00FAjxsg1BZT

Reply:
I think a better is change the type of "Boolean" to `boolean` in this line https://github.com/Azure/azure-sdk-for-java/blob/5bc550c9a5de4f8ee93a5b3141500ee34be3850d/sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/filter/UserPrincipalManager.java#L53

Reply:
Yes, will use boolean.